applications/fishing-map/public/locales/source/translations.json

```diff
@@ -23,6 +23,7 @@
     "fishing": "Fishing"
```

Comment:
source path is the fallback for the translations?

Reply:
Yes it should